recipe/meta.yaml

```diff
@@ -8,9 +8,11 @@ package:
 source:
   url: https://pypi.io/packages/source/p/psutil/psutil-{{ version }}.tar.gz
   sha256: {{ sha256 }}
+  patches:
+    - 0001-Add-missing-includes-for-macOS.patch
```

Comment:
Yeah would just download that patch into the `recipe` directory, add it to this list, and commit those changes together

Reply:
okay I'll give it a shot